### DISCUSSION TITLE
OpenVPN client ccd file error

### DISCUSSION THREAD

**Discussion by liasom** [pivpn/pivpn#1613] [Bug Reports]
## In raising this issue, I confirm the following:

`{please fill the checkboxes, e.g: [X]}`

- [X] I have read the [documentation](https://docs.pivpn.io)
- [] Is it a feature request? please consider opening a [Discussion] (https://github.com/pivpn/pivpn/discussions/new?category=feature-requests)
- [ ] Is this a support request? please consider copying the content of this document and using it to open a [Discussion](https://github.com/pivpn/pivpn/discussions/new?category=support)
- [X] I have read and understood the [PiVPN General Guidelines](https://github.com/pivpn/pivpn/#general-guidelines)
- [X] The issue I am reporting can be *replicated*.  
- [ ] The issue I am reporting *is* directly related to the pivpn installer script.
- [ ] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pivpn/pivpn/wiki/FAQ), [closed issues](https://github.com/pivpn/pivpn/issues?q=is%3Aissue+sort%3Aupdated-desc+is%3Aclosed), and [open issues](https://github.com/pivpn/pivpn/issues?q=is%3Aissue+sort%3Aupdated-desc+is%3Aopen)).

### Has your install failed? 
```
no
```

### Describe the issue
New installations (today and yesterday) of OpenVPN on two separate Raspberry Pi computers. One is Pi Zero 1.3 with 32-bit Raspberry Pi OS lite and the other is Pi Zero 2 W with Pi OS 64-bit. This error happens on both. When I create a client ovpn profile the corresponding ccd file contains an error in having no space between the ip address and subnet. This content pasted in from one of the ccd files.

ifconfig-push 10.0.8.2255.255.255.0

An older installation on Pi 4 from July does not have this issue. These two and the July installation were accomplished via Method 1 (standard). No errors during any of the installations.

**Expected behavior**
ifconfig-push 10.0.8.2 255.255.255.0

**Screenshots**

### Can you replicate the issue? Describe the steps below
Yes. Clean install today after discovering the issue yesterday.

### Have you searched for similar issues and solutions?
    Yes

### Have you taken any steps towards solving your issue?
```
If I edit the ccd file and add a space inbetween the ip address and subnet all seems to function normally.
```

#### What type of hardware are you running PiVPN at?

```
Raspberrypi Pi Zero 1.3 (yesterday) and Pi Zero 2W (today)
Responses that follow are for the Pi Zero 2W

```
#### Output of `uname -a`
```
  Linux NMA0002 5.15.61-v8+ #1579 SMP PREEMPT Fri Aug 26 11:16:44 BST 2022 aarch64 GNU/Linux
```

#### Output of `cat /etc/os-release`
```
PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
NAME="Debian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"

```

**Comment by liasom** [comment]
An older installation on Pi 4 from July does not have this issue.

**Reply by liasom** [reply]
These two and the July installation were accomplished via Method 1 (standard). No errors during any of the installations.

**Reply by coolapso** [reply]
this could have been introduced with the latest release, due to code refactoring. will investigate.

**Reply by coolapso** [reply]
could you please edit the original post and add this information please: https://raw.githubusercontent.com/pivpn/pivpn/master/.github/ISSUE_TEMPLATE/pivpn-report.md

**Reply by liasom** [reply]
Done to the best of my abilities.

**Reply by coolapso** [reply]
Cool thanks a lot, found the issue will be pushing fix soon.

**Reply by coolapso** [reply]
pushed fix to test branch, will push it to master soon.

**Comment by coolapso** [comment]
fixed. 

To update PiVPN Scripts

```
sudo su -
cd /opt/pivpn
git config pull.rebase true
git fetch
git pull origin master
```

**Reply by liasom** [reply]
Thanks. Updated the scripts and problem solved.
